pkg/trace/obfuscate/sql_plan_test.go

```diff
@@ -0,0 +1,89 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package obfuscate
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type execPlanTestCase struct {
+	// Rdbms is mysql or postgres
+	Rdbms          string                 `json:"rdbms"`
```

Comment:
nit: Dbms is more correct. Mongodb has a query planner and is non relational.

Reply:
Also nit: isn't the convention in go that all acronyms are uppercase?

Reply:
good point, updated